webkms/.htaccess

```diff
@@ -0,0 +1,15 @@
+# WebKMS Context
+#
+# homepage:
+#   https://digitalbazaar.github.io/webkms-context/
+# maintainers:
+#   @msporny
+#   @dlongley
+#   @dmitrizagidulin
```

Comment:
Probably best to keep maintainers in just one place.  Either here or the README.  They already don't sync up since I'm not in this list.  Similar with zcap files.

Reply:
Got it. Removed the READMEs, you're right.